workers/unity/Packages/io.improbable.gdk.buildsystem/Util/CommandlineParser.cs

```diff
@@ -34,8 +34,6 @@ internal static List<BuildTarget> GetBuildTargetFilter(CommandLineArgs args)
                             return BuildTarget.Android;
                         case "ios":
                             return BuildTarget.iOS;
-                        case "winx86":
-                            return BuildTarget.StandaloneWindows;
```

Comment:
Thoughts on throwing an exception saying this is no longer supported and to use the "win" target instead of falling through to the general case saying it's an unknown build target?

Reply:
In this scenario, would we treat x86 specially forever or would it drop down to a standard "unknown target" after a few releases?

Reply:
Yep, I think that would be better than the `unknown` error message

Reply:
> In this scenario, would we treat x86 specially forever or would it drop down to a standard "unknown target" after a few releases?

I imagine sometime in the future it can drop down to the 'unknown target' option. Maybe the next time we touch this code 😉

Reply:
SGTM